config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -209,3 +209,61 @@ presubmits:
       testgrid-tab-name: pr-e2e
       testgrid-alert-email: k8s-testing-clusterapi-vsphere+alerts@groups.vmware.com
       description: Runs e2e tests
+
+postsubmits:
+  kubernetes-sigs/cluster-api-provider-vsphere:
+
+  # Deploys images and binaries after all merges to master
+  - name: post-cluster-api-provider-vsphere-deploy
+    labels:
+      preset-dind-enabled: "true"
+      preset-cluster-api-provider-vsphere-e2e-creds: "true"
+    branches:
+    - ^master$
```

Comment:
I'm wondering if we want to use a `run_if_changed` on this one to only build/push new images if something meaningful has changed. Do want to push images if it was only a doc change?

Reply:
Hi @codenrhoden,

Yes, we should use a `run_if_changed`. Check [here](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/426#issuecomment-512945628) for an example.

Reply:
Ah yes, I forgot you had already provided that. You are always one step ahead! Added it. =)